### DISCUSSION TITLE
Do you have a plan to upgrade the e_SqlCipher to latest version 3.44.2 based on SQLCipher v4.5.6 released by Zetetic on Jan 18 2024?

### DISCUSSION THREAD

**Discussion by srichowdary-b** [ericsink/SQLitePCL.raw#591] [Q&A]
Do you have a plan to upgrade the e_SqlCipher to latest version 3.44.2 based on SQLCipher v4.5.6 released by Zetetic on Jan 18 2024?

**Comment by ericsink** [comment]
I don't have a plan for that (yet?).

https://github.com/ericsink/SQLitePCL.raw/wiki/Expectation-setting-for-e_sqlcipher
